cmake/LYTestWrappers.cmake

```diff
@@ -17,7 +17,7 @@ set(LY_GOOGLETEST_EXTRA_PARAMS CACHE STRING "Allows injection of additional opti
 
 find_package(Python REQUIRED MODULE)
 
-ly_set(LY_PYTEST_EXECUTABLE ${LY_PYTHON_CMD} -B -m pytest -v --tb=short --show-capture=log -c ${LY_ROOT_FOLDER}/pytest.ini --build-directory "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/$<CONFIG>")
+ly_set(LY_PYTEST_EXECUTABLE ${LY_PYTHON_CMD} -B -m pytest -v --tb=short --show-capture=stdout -c ${LY_ROOT_FOLDER}/pytest.ini --build-directory "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/$<CONFIG>")
```

Comment:
How does the `stdout` logging level correspond to the python logger categories?
Does it only output `print` calls and output to the stdout descriptor now?
Or is it outputting text above some log level threshold?

Reply:
This change accepts all logging towards stdout, which can be custom set in pytest. In LyTestTools specifically, we set stdout to INFO level only while we log DEBUG level to file.
`global _stream_handler
    if _stream_handler is None:
        _stream_handler = logging.StreamHandler(stream=sys.stdout)
        _stream_handler.setLevel(logging.INFO)
        _stream_handler.setFormatter(formatter)
        log.addHandler(_stream_handler)

    global _debug_file_handler
    if _debug_file_handler is None:
        _debug_file_handler = logging.FileHandler(debug_log_path)
        _debug_file_handler.setLevel(logging.DEBUG)
        _debug_file_handler.setFormatter(formatter)
        log.addHandler(_debug_file_handler)`

Reply:
Thanks for the clarification.